package.json

```diff
@@ -11,7 +11,8 @@
     "test": "test",
     "check": "npm run lint && prettier --check .",
     "lint": "eslint --fix ./src ./test",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "setup-resources": "node ./src/setup/setup.js"
   },
   "engines": {
     "node": ">= 14",
```

Comment:
I would use node >=16 for this project, also in the docker files too.

Reply:
The reason I used 14 is for AWS Lambda only 14 is supported: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

We might not be using Lambda, but better to have this option since this is OS.